package.json

```diff
@@ -47,6 +47,7 @@
 		"svelte": "^4.2.2",
 		"svelte-check": "^3.6.0",
 		"svelte-multiselect": "^10.2.0",
+		"svelte-tiny-slider": "^1.1.5",
```

Comment:
We already have `Carousel` from `flowbite-svelte`

Is there a reason to add another lib?

Reply:
I tried to use Carousel but it required an image or this component does not use image
that's why i add this one